integration/ca.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why upload SSH keys here?

Reply:
This is only used to test the transport with TLS, I can remove this if you think it is unnecessary